runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -242,7 +242,7 @@ public void syncObjectUnsafe(long timestamp) {
         // and it belongs to this thread use that
         if (optimisticallyOwnedByThreadUnsafe()) {
             // If there are no updates, ensure we are at the right snapshot
-            if (optimisticStream.pos() == Address.NEVER_READ) {
+            if (optimisticStream.pos() == Address.NON_ADDRESS) {
```

Comment:
Can we move to NON_ADDRESS change to another PR, I don't agree with this because the constant is being used in a lot of places where it means precisely that: that the stream/object/etc has never been read.